client/constants/QUEUE_CONFIG.json

```
@@ -25,6 +25,7 @@
   "VHA_PO_COMPLETED_TASKS_TAB_NAME": "po_completed",
   "EDUCATION_UNASSIGNED_TASKS_TAB_NAME": "education_unassigned",
   "EDUCATION_ASSIGNED_TASKS_TAB_NAME": "education_assigned",
+  "EDUCATION_COMPLETED_TASKS_TAB_NAME": "education_completed",
```

```suggestion
  "EDUCATION_EMO_COMPLETED_TASKS_TAB_NAME": "education_emo_completed",
```
@AimanK we've decided to go with naming these specific to the relevant organizations